### DISCUSSION TITLE
I want to use elFinder for my business.

### DISCUSSION THREAD

**Discussion by saptam-dev** [Studio-42/elFinder#3587] [Q&A] [answered]
Hi @nao-pon ,
Firstly I want to congratulation you for elFinder. It's really a great software.
With your permission I want to use it for my business. Can I use it?

Regards,
Saptam Dev

**Comment by nao-pon** [comment, marked as answer]
@saptam-dev Yes, elFinder is issued under a 3-clauses BSD license. Please follow that license. In addition, we will be encouraged if you can donate. Good luck! 👍 

- https://github.com/Studio-42/elFinder/blob/master/LICENSE.md

**Reply by saptam-dev** [reply]
@nao-pon OK. Thank you very much for the reply.
